benchadapt/python/benchadapt/result.py

```diff
@@ -57,14 +67,11 @@ class BenchmarkResult:
 
         Advanced: if you have a locally cloned repo, you may explicitly supply ``None``
         to this argument and its information will be scraped from the cloned repo.
-    error : Dict[str, Any]
-        A dict containing information about errors raised when running the benchmark. Any
-        schema is acceptable, but may contain stderr, a traceback, etc.
 
-    Fields one of which must be supplied, the other of which will not be posted to conbench:
+    Fields one of which must be supplied:
 
-    - ``machine_info`` (generated by default) or ``cluster_info``
-    - ``stats`` or ``error``
+    - ``machine_info`` (generated by default) xor ``cluster_info``
+    - ``stats`` and/or ``error``
```

Comment:
If we've got `xor` there, `or` is sufficient here, yeah? Or super fancy and do `∨`?

Reply:
Since I took a formal logic course I'm never sure if normal people read `or` as inclusive. Our users yes, probably?

Reply:
Yeah, the ambiguity is killer — though having xor _one line above_ should be a good helper?